pkg/reconciler/autoscaling/kpa/scaler_test.go

```diff
@@ -105,6 +108,7 @@ func TestScaler(t *testing.T) {
 		label:         "does not scale while activating",
 		startReplicas: 1,
 		scaleTo:       0,
+		wantReplicas:  -1,
```

Comment:
this one feels a bit weird. is it what we want?

Reply:
it is a signal we use when we have no data.
It is a bit weird, but oh well.